config/initializers/phonelib.rb

```diff
@@ -0,0 +1,3 @@
+Phonelib.default_country = "SG"
+Phonelib.parse_special = true
```

Comment:
whats the parse_special do?

Reply:
To use the ability to parse special numbers (Short Codes, Emergency etc.) 
According to the https://github.com/daddyz/phonelib